pom.xml

```diff
@@ -78,6 +78,18 @@
 			<version>6.0.53</version>
 			<scope>provided</scope>
 		</dependency>
+		<dependency>
+			<groupId>javax</groupId>
```

Comment:
co robi ta zależność? bo dodajesz ją ze scope provided, więc ona gdzieś powinna być w środowisku jak to jest niepotrzebne to wywal